workloads/burn-in/launch_experiment_slurm.sh

```diff
@@ -21,15 +22,20 @@
 ###    When all the jobs are down, run the verify script, put the results in the results directory
 ### - Set the full expdir from this script
 
+export HPL_DIR=${HPL_DIR:-$(cd $(dirname $0) && pwd)} # The shared directory where scripts, data, and results are stored
```

Comment:
We said that the default is set to `~/` but this is setting the default to ~/deepops/workloads/burn-in`.

Not sure what we want it to be, but the documentation said that HPL_DIR is the shared directory (I would guess ~/) and HPL_SCRIPTS_DIR is the location of the scripts (I would guss $HPL_DIR/deepops/workloads/burn-in/)

We should make this consistent and assume that the full deepops folder structure is always present.

Reply:
If we said ~/ was the default, that is wrong.  The default directory should be the location of the script.